src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -5,54 +5,97 @@ import {
- * @property {string} [rootPath = null] Prefix path to use if the first level is not the root level.
- * @property {function(object[]):object[]} [transformItems] Function to transform the items passed to the templates.
- *
- * You can also use a sort function that behaves like the standard Javascript [compareFunction](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#Syntax).
- */
-
-/**
- * @typedef {Object} BreadcrumbRenderingOptions
- * @property {function(item.value): string} createURL Creates an url for the next state for a clicked item. The special value `null` is used for the `Home` (or root) item of the breadcrumb and will return an empty array.
- * @property {BreadcrumbItem[]} items Values to be rendered.
- * @property {function(item.value)} refine Sets the path of the hierarchical filter and triggers a new search.
- * @property {Object} widgetParams All original `CustomBreadcrumbWidgetOptions` forwarded to the `renderFn`.
- */
-
-/**
- * **Breadcrumb** connector provides the logic to build a custom widget
- * that will give the user the ability to see the current path in a hierarchical facet.
- *
- * This is commonly used in websites that have a large amount of content organized in a hierarchical manner (usually e-commerce websites).
- * @type {Connector}
- * @param {function(BreadcrumbRenderingOptions, boolean)} renderFn Rendering function for the custom **Breadcrumb* widget.
- * @param {function} unmountFn Unmount function called when the widget is disposed.
- * @return {function(CustomBreadcrumbWidgetOptions)} Re-usable widget factory for a custom **Breadcrumb** widget.
- */
-export default function connectBreadcrumb(renderFn, unmountFn = noop) {
+export type BreadcrumbConnectorParamsItem = {
+  /**
+   * Label of the category or subcategory.
+   */
+  label: string;
+
+  /**
+   * Value of breadcrumb item.
+   */
+  value: string;
+};
+
+export type BreadcrumbConnectorParams = {
+  /**
+   * Attributes to use to generate the hierarchy of the breadcrumb.
+   */
+  attributes: string[];
+
+  /**
+   * Prefix path to use if the first level is not the root level.
+   */
+  rootPath?: string;
+
+  /**
+   * Function to transform the items passed to the templates.
+   */
+  transformItems?: TransformItems<BreadcrumbConnectorParamsItem>;
+
+  /**
+   * The level separator used in the records.
+   *
+   * @default '>'
+   */
+  separator?: string;
+};
+
+export type BreadcrumbRendererOptions = {
+  /**
+   * Creates the URL for a single item name in the list.
+   */
+  createURL: CreateURL<BreadcrumbConnectorParamsItem['value']>;
+
+  /**
+   * Array of objects defining the different values and labels.
+   */
+  items: BreadcrumbConnectorParamsItem[];
+
+  /**
+   * Sets the path of the hierarchical filter and triggers a new search.
+   */
+  refine: (value: BreadcrumbConnectorParamsItem['value']) => void;
+
+  /**
+   * @internal
+   */
+  canRefine: boolean;
```

Comment:
Seems that canRefine property exists somewhere, but is not public. Do you know where it comes from? And why is used?